File: Latihan1B-cpp/1b.cpp
#include <iostream>
#include <string>

using namespace std;

class DaftarTim{

    private:

    string namaTim;
    string negaraAsal;
    int tahunBerdiri;
    string pemain;
    string pemain2;
    int nomor;
    int nomor2;

    public:

    DaftarTim(){
    }

    DaftarTim(string namaTim, string negaraAsal, int tahunBerdiri, string pemain, int nomor, string pemain2, int nomor2){
        this->namaTim = namaTim;
        this->negaraAsal = negaraAsal;
        this->tahunBerdiri = tahunBerdiri;
        this->pemain = pemain;
        this->pemain2 = pemain2;
        this->nomor = nomor;
        this->nomor2 = nomor2;
    }

    // Nama Tim
    void setNamaTim(string namaTim){
        this->namaTim = namaTim;
    }

    string getNamaTim(){
        return this->namaTim;
    }

    // Negara Asal
    void setNegaraAsal(string negaraAsal){
        this->negaraAsal = negaraAsal;
    }

    string getNegaraAsal(){
        return this->negaraAsal;
    }

    // Tahun Berdiri
    void setTahunBerdiri(int tahunBerdiri){
        this->tahunBerdiri = tahunBerdiri;
    }

    int getTahunBerdiri(){
        return this->tahunBerdiri;
    }

    // Pemain
    void setPemain(string pemain){
        this->pemain = pemain;
    }

    string getPemain(){
        return this->pemain;
    }

    void setPemain2(string pemain2){
        this->pemain2 = pemain2;
    }

    string getPemain2(){
        return this->pemain2;
    }

    // Nomor Punggung
    void setNomor(int nomor){
        this->nomor = nomor;
    }

    int getNomor(){
        return this->nomor;
    }

    void setNomor2(int nomor2){
        this->nomor2 = nomor2;
    }

    int getNomor2(){
        return this->nomor2;
    }

    ~DaftarTim(){
    }

};

File: Latihan1B/1b.php
<?php

// Buat program untuk membuat daftar tim sepakbola yang memiliki atribut 
// nama tim, negara asal tim, tahun berdiri tim, serta pemain, menggunakan 
// setiap bahasa (C++, PHP, Python dan Java)!

class DaftarTim
{
    private $namaTim = "";
    private $negaraAsal = "";
    private $tahunBerdiri = 0;
    private $pemain = "";
    private $pemain2 = "";
    private $nomor = 0;
    private $nomor2 = 0;

    public function __construct($namaTim = "", $negaraAsal = "", $tahunBerdiri = 0, 
        $pemain = "", $nomor = 0, $pemain2 = "", $nomor2 = 0){
        $this->namaTim = $namaTim;
        $this->negaraAsal = $negaraAsal;
        $this->tahunBerdiri = $tahunBerdiri;
        $this->pemain = $pemain;
        $this->nomor = $nomor;
        $this->pemain2 = $pemain2;
        $this->nomor2 = $nomor2;
    }

    // Nama Tim
    public function setNamaTim($namaTim){
        $this->namaTim = $namaTim;
    }

    public function getNamaTim(){
        return $this->namaTim;
    }

    // Negara Asal
    public function setNegaraAsal($negaraAsal){
        $this->negaraAsal = $negaraAsal;
    }

    public function getNegaraAsal(){
        return $this->negaraAsal;
    }

    // Tahun Berdiri
    public function setTahunBerdiri($tahunBerdiri){
        $this->tahunBerdiri = $tahunBerdiri;
    }

    public function getTahunBerdiri(){
        return $this->tahunBerdiri;
    }

    // Pemain
    public function setPemain($pemain){
        $this->pemain = $pemain;
    }

    public function getPemain(){
        return $this->pemain;
    }

    public function setPemain2($pemain2){
        $this->pemain2 = $pemain2;
    }

    public function getPemain2(){
        return $this->pemain2;
    }

    // Nomor Punggung
    public function setNomor($nomor){
        $this->nomor = $nomor;
    }

    public function getNomor(){
        return $this->nomor;
    }

    public function setNomor2($nomor2){
        $this->nomor2 = $nomor2;
    }

    public function getNomor2(){
        return $this->nomor2;
    }

}

?>